src/utils.ts

```diff
@@ -0,0 +1,34 @@
+const stringToBoolean = (value: string) => {
+  const stringBooleans = ["true", "false"];
+  if (value && stringBooleans.includes(value)) {
+    return value === "true" ? true : false;
+  }
+  return value;
+};
+
+const stringToNumber = (value: string) => {
+  const number = Number(value);
+  if (isNaN(number)) return value;
+  return number;
+};
+
+export const parseUserData = (data: any) => {
+  // Booleans
+  const isSubscribers = data?.user?.plan?.isSubscribers;
+  if (isSubscribers && typeof isSubscribers === "string") {
+    data.user.plan.isSubscribers = stringToBoolean(isSubscribers);
```

Comment:
Oh! It is modifying the same input object.

In my opinion, it could be dangerous and less intuitive. What do you think about create a new object `UserData` as output of `parseUserData`?